packages/@aws-cdk/aws-events-targets/package.json

```diff
@@ -1,110 +1,111 @@
 {
-  "name": "@aws-cdk/aws-events-targets",
```

Comment:
Any reason this changed? tabs/spaces?

Reply:
The only change I expected to see there was having the "@aws-cdk/aws-sqs": "^0.32.0" in the dependencies... 🤔I think I can make a revert and try if everything works fine